DEVELOPER.md

```diff
@@ -14,10 +14,12 @@ See http://kafka.apache.org/documentation.html#consumerconfigs for details about
 
     input {
         kafka {
-            topic_id => ... # string (required), The topic to consume messages from
+            topic_id => ... # string (required), The topic to consume messages from. Can be a java regular expression for whitelist of topics.
```

Comment:
we could but then it loses some backwards compatibility with anyone that already has configs.